scripts/foundry/contract-upgrades/2.1.0/README.md

```diff
@@ -0,0 +1,51 @@
+# Nitro contracts 2.1.0 upgrade
```

Comment:
Also need a new section in https://github.com/OffchainLabs/orbit-actions/blob/main/README.md to mention the arbos upgrade

Reply:
and mention nitro-node v3.1.0 is required

Reply:
https://github.com/OffchainLabs/orbit-actions/pull/16/commits/a7776667f4be013962aa05f6bc94bb7d7d48ebca